demos/fleet_provisioning/fleet_provisioning_with_csr/core_mqtt_config.h

```diff
@@ -41,7 +41,7 @@
 #endif
 
 #ifndef LIBRARY_LOG_LEVEL
-    #define LIBRARY_LOG_LEVEL    LOG_INFO
+    #define LIBRARY_LOG_LEVEL    LOG_WARN
```

Comment:
Any specific reason for the change of logging level for coreMQTT? Just to reduce noise in the demo logs coming from coreMQTT?

Reply:
Yes, too much noise